src/main/java/org/sonar/plugins/stash/StashRequestFacade.java

```diff
@@ -138,42 +138,52 @@ public void postCommentPerIssue(String project, String repository, String pullRe
       
       // Severity available to create a task
       List<String> taskSeverities = getReportedSeverities();
-      
+
+      // Let's call this "issue_loop"
       for (SonarQubeIssue issue : issueReport.getIssues()) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS135)